kolibri/plugins/coach/assets/src/views/exams/ExamReportPage/ManageExamModals.vue

```diff
@@ -149,6 +151,8 @@
     $trs: {
       changeExamStatusTitle: 'Change exam status',
       changeExamStatusDescription: 'Learners can only see active exams',
+      changeExamStatusActive: 'Active',
```

Comment:
I really wish there was a way for us to provide more context to translators from our json files - this makes it possible to fix, but still not clear that it should be appropriately gendered!

Reply:
@indirectlylit @rtibbles You were too fast for me here, and I'm afraid that I don't understand what exactly was "fixed" here... Is it possible to get a screenshot or similar?

"Aligning experiences" is a worthy goal, but in the line 154 I still see the adjectival form `Active`... :confused: 

My recommendation was to switch to verb, `Activate/Deactivate` which would avoid the whole mess, is there a reason why that could not be implemented?

Reply:
> I don't understand what exactly was "fixed" here... 

Previously there were only two strings (`activeOption` and `inactiveOption`) that had to be used with both exams and lessons.

Now there are four strings, so they can be customized to the modified noun:

* `changeLessonStatusActive`: Activas
* `changeLessonStatusInctive`: Inactivas
* `changeExamStatusActive`: Activos
* `changeExamStatusInctive`: Inactivos

> Is it possible to get a screenshot or similar?

the screenshots would look exactly the same because the Lesson and Exam forms are the same in English

> My recommendation was to switch to verb

I can do that too, but usually we reserve verbs for buttons so I was hesitant to...

Can still do that if you think it's better - I would loop @jtamiace in on that

Reply:
> Now there are four strings

Now I get it, thanks... No need for screenshots! :+1: 
This resolves the immediate concern, the only thing that translators would need is a bit more context, so we should put back on the table the discussion about solving the context-on-Crowdin issue as soon as this release is done.

I also understand the hesitation, and the will to stick to `button=verb` convention, and now I'm hesitant too (we do demand action, but still... :thinking: )